Production/python/Production.py

```diff
@@ -92,7 +92,7 @@
     process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
Few things/questions  which I've just noticed, not fully related to the changes:
- Would it makes sense to add v2.5 for `isRun2UL` case similarly to `isRun3`? I see that in `env.sh` they both are set up with 12.4.X  so v2.5 should be runnable in this mode too https://github.com/cms-tau-pog/TauMLTools/blob/04a6aab8b160a083d5b5c50686d8e120fc60dee7/Production/python/Production.py#L148-L158
- I think we can drop now `isRun2PreUL` flag ([L52](https://github.com/cms-tau-pog/TauMLTools/blob/04a6aab8b160a083d5b5c50686d8e120fc60dee7/Production/python/Production.py#L52), [L319](https://github.com/cms-tau-pog/TauMLTools/blob/04a6aab8b160a083d5b5c50686d8e120fc60dee7/Production/python/Production.py#L139))
- I am not sure that these lines are needed, the Path for `isPhase2` already includes `slimmedElectronsMerged` right?: https://github.com/cms-tau-pog/TauMLTools/blob/04a6aab8b160a083d5b5c50686d8e120fc60dee7/Production/python/Production.py#L316-L317
- About the `else` statements in [L159](https://github.com/cms-tau-pog/TauMLTools/blob/04a6aab8b160a083d5b5c50686d8e120fc60dee7/Production/python/Production.py#L159) and [L295](https://github.com/cms-tau-pog/TauMLTools/blob/04a6aab8b160a083d5b5c50686d8e120fc60dee7/Production/python/Production.py#L295): does it basically target now only embedded samples and data (since we drop preUL support)? Would it make sense to make it explicit and raise exception otherwise? 
- Is the `boostedSequence` needed now or having a common for all eras `boostedTaus_InputTag = cms.InputTag('slimmedTausBoosted')` is ok?

Reply:
done